Investigate failing postpublish script

When I published v0.0.4, I saw the following error:

```
> d3-tile@0.0.4 postpublish .
> VERSION=`node -e 'console.log(require("./package.json").version)'`; git push && git push --tags && cp build/d3-tile.js ../d3.github.com/d3-tile.v0.0.js && cp build/d3-tile.min.js ../d3.github.com/d3-tile.v0.0.min.js && cd ../d3.github.com && git add d3-tile.v0.0.js d3-tile.v0.0.min.js && git commit -m "d3-tile ${VERSION}" && git push && cd - && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js

Counting objects: 3, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 285 bytes | 0 bytes/s, done.
Total 3 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To git@github.com:d3/d3-tile.git
   30c7c13..393a87c  master -> master
Counting objects: 1, done.
Writing objects: 100% (1/1), 156 bytes | 0 bytes/s, done.
Total 1 (delta 0), reused 0 (delta 0)
To git@github.com:d3/d3-tile.git
 * [new tag]         v0.0.4 -> v0.0.4
cp: cannot create regular file '../d3.github.com/d3-tile.v0.0.js': No such file or directory

npm ERR! Linux 4.4.0-62-generic
npm ERR! argv "/home/curran/.nvm/versions/node/v7.4.0/bin/node" "/home/curran/.nvm/versions/node/v7.4.0/bin/npm" "publish"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! d3-tile@0.0.4 postpublish: `VERSION=`node -e 'console.log(require("./package.json").version)'`; git push && git push --tags && cp build/d3-tile.js ../d3.github.com/d3-tile.v0.0.js && cp build/d3-tile.min.js ../d3.github.com/d3-tile.v0.0.min.js && cd ../d3.github.com && git add d3-tile.v0.0.js d3-tile.v0.0.min.js && git commit -m "d3-tile ${VERSION}" && git push && cd - && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the d3-tile@0.0.4 postpublish script 'VERSION=`node -e 'console.log(require("./package.json").version)'`; git push && git push --tags && cp build/d3-tile.js ../d3.github.com/d3-tile.v0.0.js && cp build/d3-tile.min.js ../d3.github.com/d3-tile.v0.0.min.js && cd ../d3.github.com && git add d3-tile.v0.0.js d3-tile.v0.0.min.js && git commit -m "d3-tile ${VERSION}" && git push && cd - && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the d3-tile package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     VERSION=`node -e 'console.log(require("./package.json").version)'`; git push && git push --tags && cp build/d3-tile.js ../d3.github.com/d3-tile.v0.0.js && cp build/d3-tile.min.js ../d3.github.com/d3-tile.v0.0.min.js && cd ../d3.github.com && git add d3-tile.v0.0.js d3-tile.v0.0.min.js && git commit -m "d3-tile ${VERSION}" && git push && cd - && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs d3-tile
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls d3-tile
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/curran/repos/d3-tile/npm-debug.log
```

Will investigate.

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "pretest": "rm -rf build && mkdir build && rollup --banner \"$(preamble)\" -g d3-array:d3 -f umd -n d3 -o build/d3-tile.js -- index.js",
     "test": "tape 'test/**/*-test.js' && eslint index.js src",
     "prepublish": "npm run test && uglifyjs --preamble \"$(preamble)\" build/d3-tile.js -c -m -o build/d3-tile.min.js",
-    "postpublish": "VERSION=`node -e 'console.log(require(\"./package.json\").version)'`; git push && git push --tags && cp build/d3-tile.js ../d3.github.com/d3-tile.v0.0.js && cp build/d3-tile.min.js ../d3.github.com/d3-tile.v0.0.min.js && cd ../d3.github.com && git add d3-tile.v0.0.js d3-tile.v0.0.min.js && git commit -m \"d3-tile ${VERSION}\" && git push && cd - && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js"
+    "postpublish": "VERSION=`node -e 'console.log(require(\"./package.json\").version)'`; git push && git push --tags && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js"
   },
   "dependencies": {
     "d3-array": "1"

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "pretest": "rm -rf build && mkdir build && rollup --banner \"$(preamble)\" -g d3-array:d3 -f umd -n d3 -o build/d3-tile.js -- index.js",
     "test": "tape 'test/**/*-test.js' && eslint index.js src",
     "prepublish": "npm run test && uglifyjs --preamble \"$(preamble)\" build/d3-tile.js -c -m -o build/d3-tile.min.js",
-    "postpublish": "VERSION=`node -e 'console.log(require(\"./package.json\").version)'`; git push && git push --tags && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js"
+    "postpublish": "git push && git push --tags && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js"
   },
   "dependencies": {
     "d3-array": "1"

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "pretest": "rm -rf build && mkdir build && rollup --banner \"$(preamble)\" -g d3-array:d3 -f umd -n d3 -o build/d3-tile.js -- index.js",
     "test": "tape 'test/**/*-test.js' && eslint index.js src",
     "prepublish": "npm run test && uglifyjs --preamble \"$(preamble)\" build/d3-tile.js -c -m -o build/d3-tile.min.js",
-    "postpublish": "git push && git push --tags && zip -j build/d3-tile.zip -- LICENSE README.md build/d3-tile.js build/d3-tile.min.js"
+    "postpublish": "git push && git push --tags"
   },
   "dependencies": {
     "d3-array": "1"

